packages/core/resources/icons/icons.json

```diff
@@ -2693,5 +2693,19 @@
     "tags": "tick mark, sign, ok, approved, success",
     "group": "action",
     "content": "\\e75f"
+  },
+  {
+    "name": "Follower",
+    "className": "pt-icon-follower",
+    "tags": "person, human, male, female, character, customer, individual, social",
+    "group": "action",
+    "content": "\\e760"
+  },
+  {
+    "name": "Following",
+    "className": "pt-icon-following",
+    "tags": "person, human, male, female, character, customer, individual, social",
+    "group": "action",
```

Comment:
i think these should be in `"interface"` with the other people:

![image](https://cloud.githubusercontent.com/assets/464822/22839333/c4c282be-ef7d-11e6-9015-36be47f34f19.png)